docs/pages/foundations/forms/composition_+_behavior.js

```diff
@@ -0,0 +1,278 @@
+                <Table.HeaderCell key={item}>
+                  <Text size="200" weight="bold">
+                    {item}
+                  </Text>
+                </Table.HeaderCell>
+              ))}
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            <Table.Row>
+              {[
+                'Title',
+                'Heading/lg',
+                '$font-size-600; $font-weight-semibold',
+                '$color-text-default',
+                'Web',
+              ].map((item) => (
+                <Table.Cell key={item}>
+                  <Text size="200">{item}</Text>
+                </Table.Cell>
+              ))}
+            </Table.Row>
+            <Table.Row>
+              {[
+                'Section title',
+                'Heading/md',
+                '$font-size-500; $font-weight-semibold',
+                '$color-text-default',
+                'Web',
+              ].map((item) => (
+                <Table.Cell key={item}>
+                  <Text size="200">{item}</Text>
+                </Table.Cell>
+              ))}
+            </Table.Row>
+            <Table.Row>
+              {[
+                'Subtext',
+                'Body/lg regular',
+                '$font-size-300; $font-weight-normal',
+                '$color-text-subtle',
+                'Web',
+              ].map((item) => (
+                <Table.Cell key={item}>
+                  <Text size="200">{item}</Text>
+                </Table.Cell>
+              ))}
+            </Table.Row>
+          </Table.Body>
+        </Table>
+      </MainSection>
+
+      <MainSection name="Form fields and controls">
+        <MainSection.Subsection description="Form fields and controls should match the text size determined by their individual components and be organized in a logical way to increase user comprehension." />
+        <MainSection name="Buttons">
+          <MainSection.Subsection
+            title="Placement"
+            description="Buttons should be at the bottom of the form.
+
+            If a form is in the context of a page, buttons should be start-aligned and in line to the form contents."
+          />
+          <Box maxWidth={570} maxHeight={586} borderStyle="sm" rounding={4}>
+            <Mask rounding={4}>
+              <Image
+                alt="A form on a page with the buttons start-aligned along with the form text and controls."
+                naturalHeight={1758}
+                naturalWidth={1710}
+                src="https://i.pinimg.com/originals/ea/2c/3b/ea2c3b5c9adc67ebfff55309f713d96e.png"
+              />
+            </Mask>
+          </Box>
+          <MainSection.Subsection description="However, if a form is in the context of a [Modal](../../web/modal) or [OverlayPanel](../../web/overlaypanel), end-aligned buttons are appropriate, in order to be consistent with placement of buttons in other overlays." />
```

Comment:
MainSection.Subsection here... not a fan... but we can refactor another time. 
Ideally. 

<Flex gap={3} direction="column">
<Text>
<Image>
...

Or markdown

Reply:
I'd like to come back and refactor this and Messages when we can have a better way to manage image widths.